sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/ReactorNettyClient.java

```diff
@@ -43,6 +47,21 @@
 
     private static final Logger logger = LoggerFactory.getLogger(ReactorNettyClient.class.getSimpleName());
 
+    private static final MethodHandle HTTP_CLIENT_WARMUP;
+
+    static {
+        MethodHandle httpClientWarmup = null;
+        try {
+            httpClientWarmup = MethodHandles.publicLookup()
+                .findVirtual(reactor.netty.http.client.HttpClient.class, "warmup", MethodType.methodType(Mono.class));
+        } catch (Throwable throwable) {
```

Comment:
Though this is nit, but catching Throwable here is catching everything. 
Shouldn't we just catch the exception which is thrown by `publicLookup` or `findVirtual` ?

Reply:
Good catch, I thought for some reason `findVirtual`'s checked exception type was `Throwable` but that is only for the `invoke` call.

Reply:
Yes, please update this.